jsapp/scss/components/_kobo.modal.scss

```diff
@@ -562,6 +566,18 @@ $z-modal-x: 10;
     &:hover .form-view__cell--translation-name .form-view__icon-button {
       opacity: 1;
       visibility: visible;
+
+      // Hide the on hover button completely if disabled
+      &:disabled {
+        opacity: 0;
+        visibility: hidden;
```

Comment:
Wouldn't `display: none` be good enough?

Reply:
Don't know if you see it on your screen but if I do this the language text that would have the icon next to it moves on hover

Reply:
Oh I see. I would say that it means that the parent styles are not good enough :P Let's do this on `form-view__cell--translation-name`:

```
display: flex;
flex-direction: row;
align-items: center;
```